setup_py3.py

```diff
@@ -153,7 +153,7 @@ def run(self):
             r'^%define sourcebundle.*$': '%define sourcebundle {0}'.format(sourcefile),
             r'^%define release.*$': '%define release {0}'.format(buildnum),
             r'^Requires:': 'Requires: Portal >= 3.0 ',
```

Comment:
we don't need (and can't have) portal 3.x in a Python 3 environment, needs to be removed